src/Common/src/System/Net/NTAuthentication.Common.cs

```diff
@@ -203,8 +202,7 @@ internal string GetOutgoingBlob(string incomingBlob)
 
         internal byte[] GetOutgoingBlob(byte[] incomingBlob, bool thrownOnError)
         {
-            SecurityStatusPal statusCode;
-            return GetOutgoingBlob(incomingBlob, thrownOnError, out statusCode);
+            return GetOutgoingBlob(incomingBlob, thrownOnError, out _);
```

Comment:
Unless there's something I don't know about `out _`, the edits to this file are style without substance, and thus this file should be dropped from the PR.

Reply:
Reverted